### PR TITLE
chore:pr-template added

### LINKED ISSUE #23: Chore -> Add PR Template

### PR DESCRIPTION
**Pull request by VenomFate-619**
The template will reflected in the next coming PRs
Fixes #23

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] 🎨 Style
+- [ ] 🧑‍💻 Code Refactor
+- [ ] 🔥 Performance Improvements
+- [ ] ✅ Test
+- [ ] 🤖 Build
+- [ ] 🔁 CI
+- [ ] 📦 Chore (Release)
+- [ ] ⏩ Revert
+
+## Added tests?
```
Comment (Neha) [inline .github/pull_request_template.md:24]: Add an option: updated test-cases

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
```
Comment (Neha) [inline .github/pull_request_template.md:3]: Also, add : 

snapshot of the test-cases that are passing

Steps to test the feature

**Review by Neha — CHANGES_REQUESTED** [review]
Thank you for the Template. This looks good. I have added 2 comments. Please check

File: .github/pull_request_template.md (hunk 2 above)
Comment (VenomFate-619) [reply .github/pull_request_template.md:3]: maybe in the coming future we will need to remove _snapshot of the test cases that are passing_, as there will be a CI running for a PR to test whether text cases are passing or not

Comment (Neha) [reply .github/pull_request_template.md:3]: @VenomFate-619 Snapshot will be required even after CI as we are expecting every dev to test the test-cases passing at their local before raising a PR

**Review by ekas — APPROVED** [review]

**Comment by ekas** [comment]
Look good to me. @Neha If you agree we could merge.

**Review by Neha — APPROVED** [review]

**Comment by ekas** [comment]
@VenomFate-619 Thank you so much for your contribution.

**Pull request merged**